content/site/about-us.md

```diff
@@ -0,0 +1,18 @@
+---
+{
+    title: "About Us",
+    description: 'About Unicorn Utterances - who we are, what our goals are, and how we want to help others learn',
+}
+---
+
+> Our goal is to provide the best resources possible to learn any computer science related topic
+
+Whether it's from how memory is allocated in assembly or how to do complex CSS animations, to what a For loop is, we want to provide resources that are welcoming, supportive, and informational.
+
+Our content will have a wide range of subject matters, difficulty curves, and (we hope) perspectives. We want to make resources for the seasoned full-time developer just as we would a newcomer hobbyist and everyone between the two.
+
+ We know this is a lofty goal, though, and we don't want to do it alone. If you're interested in helping out, open a pull request and help us reach others by providing translations, help the code maintenance on this site, write a post for us, or jump in on discussions and help other readers.
+ 
+ If you’d like to get in touch with us, our GitHub is probably the best way to do so.
```

Comment:
You might want to actually add email / contact links somewhere on the page if you're mentioning it here, as I don't think there are any other references to the GitHub org other than the "view on GH" link at the bottom of the articles.

Maybe a small footer on each page linking to the about section, github, email, social media, etc. would do the trick.

Reply:
I would love to add social media and email but we currently don't have either. I will add actual links to what I was talking about though